src/Analyzers/Analyzers/test/TestReferences.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.Extensions.DependencyModel;
+using Microsoft.Extensions.DependencyModel.Resolution;
+
+namespace Microsoft.AspNetCore.Analyzers;
+
+public static class TestReferences
+{
+    public static readonly ReferenceAssemblies ReferenceAssemblies = new("some-tfm");
```

Comment:
💡 If you have a known correct TFM for testing, it would be better to use it here.

Reply:
It's a little tricky, because the right way to do this is read the value of an msbuild property. While do-able, it seems like a lot of work for something that doesn't appear to be important. I'll leave this be for now.